e2e/tests/omnibus-sepolia.toml/LegacyMarket.e2e.js

```diff
@@ -0,0 +1,117 @@
+const assert = require('assert');
+const { migrateLegacyMarket } = require('../../tasks/migrateLegacyMarket');
+const { getLpPosition } = require('../../tasks/getLpPosition');
+const { getAccountCollateral } = require('../../tasks/getAccountCollateral');
+const { ethers } = require('ethers');
+
+const log = require('debug')(`e2e:${require('path').basename(__filename, '.js')}`);
+
+const run = function () {
+  let LegacyMarketProxy;
+  let CoreProxy;
+  let V2xSnx;
+
+  // some big wallets
+  // https://etherscan.io/token/0x89FCb32F29e509cc42d0C8b6f058C993013A843F#balances
```

Comment:
```suggestion
  // https://sepolia.etherscan.io/token/ ??? token address? ??? #balances
```

Reply:
the addresses here are actually mainnet so I dont think its necessary to change the url here to something less useful?

Reply:
it is a sepolia test so we need to either use legit addresses from sepolia, or remove this part of test entirely as it is functional